vendor.conf

```diff
@@ -55,9 +55,9 @@ github.com/vishvananda/netlink b2de5d10e38ecce8607e6b438b6d174f389a004e
 github.com/BurntSushi/toml a368813c5e648fee92e5f6c30e3944ff9d5e8895
 github.com/samuel/go-zookeeper d0e0d8e11f318e000a8cc434616d69e329edc374
 github.com/deckarep/golang-set ef32fa3046d9f249d399f98ebaf9be944430fd1d
-github.com/coreos/etcd v3.2.1
+github.com/coreos/etcd v3.3.9
```

Comment:
Is it wise to update to 3.3 here?
Swarmkit still only has 3.2.1.

Reply:
Yeah, we discussed this with @thaJeztah not sure either :sweat: 
I can remove the bump from there and try to update to latest `etcd` in `swarmkit` and `moby` :angel: